app/assets/javascripts/calendars/guiders-appointments.es6

```diff
@@ -30,12 +30,40 @@
 
       super(el, calendarConfig);
 
+      this.alterHeight();
+      $(window).on('resize', this.debounce(this.alterHeight.bind(this), 20));
+
       this.eventChanges = [];
       this.actionPanel = $('[data-action-panel]');
 
       this.setupUndo();
     }
 
+    debounce(func, wait, immediate) {
```

Comment:
Feels like this could be promoted up but I guess let's see how much it gets used